doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -43,7 +43,10 @@ Common
 
 The following changes are relevant for all device families.
 
-There are no entries for this section yet.
+Common Application Framework (CAF)
+----------------------------------
+
+* Enable power management in modules by default if only kconfig option :kconfig:`CAF_PM_EVENTS` is enabled.
```

Comment:
@greg-fer , do techwriters review it now or at release?

Reply:
Now, to have a faster turnaround around the release.